assets/packages/adminbasics/build/adminbasics.js

```diff
@@ -29829,6 +29832,9 @@
 
         $("div", this).html(pValue + "%");
       });
+      /* set default for select2 */
+
+      $.fn.select2.defaults.set("theme", "bootstrap");
```

Comment:
If already done as default, why doing it again now?

Reply:
> If already done as default, why doing it again now?

Where it's done ? Not in `adminbasics.js` ?

Reply:
On globalMethods.
https://github.com/LimeSurvey/LimeSurvey/pull/3012/files#diff-ede93d7c4e63dba99f44a55b638edd64610bd2096268abc5375ba0bc21ffa149R153

Makes sense what I am saying?

Reply:
**build**/adminbasics.js

This part came form globalMethods … it's build part …

Reply:
I **never** update build part manually

Reply:
Ohh didn't notice it was build...!